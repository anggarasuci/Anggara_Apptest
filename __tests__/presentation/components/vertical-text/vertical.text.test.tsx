import React from 'react';
import {render} from '@testing-library/react-native';
import VerticalTextView from '../../../../src/presentation/components/vertical-text/vertical.text';

test('renders VerticalTextView correctly', () => {
  const {toJSON} = render(<VerticalTextView label="Label" value="Value" />);

  expect(toJSON()).toMatchSnapshot();
});
