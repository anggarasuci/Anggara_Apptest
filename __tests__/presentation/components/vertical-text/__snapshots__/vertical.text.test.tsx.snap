// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders VerticalTextView correctly 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <Text
    style={
      {
        "color": "#9E9E9E",
        "fontSize": 12,
        "fontWeight": "bold",
      }
    }
  >
    Label
  </Text>
  <Text
    style={
      {
        "color": "#404040",
        "fontSize": 18,
        "fontWeight": "normal",
      }
    }
  >
    Value
  </Text>
</View>
`;
